triangle_area.rb

```diff
@@ -0,0 +1,5 @@
+print 'Enter the base of the triangle(a): '
+base = gets.chomp.to_f
+print 'Enter the height of the triangle(h): '
+height = gets.chomp.to_f
+print "The area of this triangle is #{base * height * 0.5}"
```

Comment:
it is correct unless we add negative numbers into the base or height.
You can add some checks here, as you did it in ideal_weight.rb